### PR TITLE
replace instances of "/etc/.pihole" with ${PI_HOLE_LOCAL_REPO}

### PR DESCRIPTION
**Pull request by jftuga**
This is my first ever pull request, hope I doing it right. :-)

See also: https://github.com/pi-hole/pi-hole/issues/1323

### CODE REVIEW

**Comment by Mcat12** [comment]
Thank you for the PR, but please make your changes in the `development` branch (or make a branch off of `development`) and then open a PR for the `development` branch. See these docs for more information: [Contributing](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md#technical-requirements), [PR Template](https://github.com/pi-hole/pi-hole/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

**Comment by Mcat12** [comment]
Ok, this should work as well.

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -669,7 +669,7 @@ installScripts() {
 }
 
 installConfigs() {
-  # Install the configs from /etc/.pihole to their various locations
+  # Install the configs from ${PI_HOLE_LOCAL_REPO} to their various locations
```
Comment (Mcat12) [inline automated install/basic-install.sh]: Shouldn't need the `${...}` in a comment.

Comment (PromoFaux) [inline automated install/basic-install.sh]: Though, that said, if the install location was ever to change, it's probably a good idea to have the variable name in it's place :)

Comment (Mcat12) [inline automated install/basic-install.sh]: Ya, I meant to say that `PI_HOLE_LOCAL_REPO` would be better in a comment than `${PI_HOLE_LOCAL_REPO}`.

Comment (dschaper) [inline automated install/basic-install.sh]: I agree, can this be changed to just have the variable name in the comment?

Comment (Mcat12) [inline automated install/basic-install.sh]: :+1:

Comment (diginc) [inline automated install/basic-install.sh]: why even comment, just put it in the echo a couple lines lower and be more descriptive:

```
    echo "::: Installing configs from ${PI_HOLE_LOCAL_REPO}..." 
```

the rest of the echos describe 'various locations'

**Pull request closed without merging**